lib/views/frontend/spree/orders/show.html.erb

```diff
@@ -0,0 +1,114 @@
+    </div>
+    <div>
+      <dt class="text-uppercase text-center text-lg-left"><%= Spree.t(:payment) %></dt>
+      <dd class="text-center text-lg-left">
+        <%= render collection: @order.payments.valid, partial: 'spree/shared/payment' %>
+      </dd>
+    </div>
+  </dl>
+  <hr class="w-100 mt-0 mb-4 d-sm-none" />
+  <div class="order-show-line-items d-flex flex-column align-items-center">
+    <% @order.line_items.each do |item| %>
+      <div class="order-show-line-items-line-item d-flex w-100">
+        <%= link_to product_image(item.variant), item.product %>
+        <div class="order-show-line-items-line-item-desc w-100">
+          <span class="order-show-line-items-line-item-desc-name">
+            <%= item.name %>
+          </span>
+          <span class="order-show-line-items-line-item-desc-subtitle text-break text-uppercase">
+            <ul class="list-unstyled mb-0">
+              <% item.variant.option_values.sort { |ov| ov.option_type.position }.each do |ov| %>
+                <li><%= "#{ov.option_type.presentation}: #{ov.name.titleize}" %></li>
+              <% end %>
+            </ul>
+          </span>
+          <div class="order-show-line-items-line-item-desc-price">
+            <div class="order-show-line-items-line-item-desc-price-attr-name d-inline d-lg-block">
+              <span><%= Spree.t(:price) %></span>
+            </div>
+            <span class="font-weight-bold"><%= item.single_money.to_html %></span>
+          </div>
+          <div class="order-show-line-items-line-item-desc-quantity">
+            <div class="order-show-line-items-line-item-desc-quantity-attr-name d-inline d-lg-block">
+              <span><%= Spree.t(:quantity) %></span>
+            </div>
+            <span class="order-show-line-items-line-item-desc-quantity-val d-lg-block"><%= item.quantity %></span>
+          </div>
+          <div class="order-show-line-items-line-item-desc-total">
+            <div class="order-show-line-items-line-item-desc-total-attr-name d-inline d-lg-block">
+              <span><%= Spree.t(:total) %></span>
+            </div>
+            <span class="font-weight-bold d-lg-block"><%= item.display_total.to_s %></span>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+  <hr class="w-100 mt-4 mb-0 d-sm-none" />
+  <div id="checkout-summary" class="w-100" data-hook="checkout_summary_box">
+    <%= render partial: 'spree/checkout/summary', locals: { order: @order } %>
+  </div>
+</div>
+
+<% intent_secrets = @order.payments.map do |payment|
+  next unless payment.intent_client_key
+  {
+    intent_key: payment.intent_client_key,
+    pk_key: payment.payment_method.preferred_publishable_key
+  }
+end.last %>
+
+<% if intent_secrets %>
+  <script type="text/javascript" src="https://js.stripe.com/v3/"></script>
+
+  <script>
+
+    function confirmCardPaymentResponseHandler(response) {
+      $.post("/intents/handle", {response: response, order: "<%= @order.number %>"})
```

Comment:
@chhga we need to include authenticity token here, like here:

https://github.com/spree/spree/blob/6e95ff6a4ff362ec8767e7f4ddcec7b2c208cf25/backend/app/assets/javascripts/spree/backend/admin.js#L278